### PR TITLE
Improve first-line detection of Toit.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: vscode/syntaxes/toit.tmLanguage.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "comment": "Don't edit the JSON file, but edit the YAML file instead. Use npx js-yaml < syntaxes/toit.tmLanguage.yaml > syntaxes/toit.tmLanguage.json Use the scope inspector from the command palette with the Developer: Inspect Editor Tokens and Scopes\n",
```
Comment (kasperl) [inline vscode/syntaxes/toit.tmLanguage.json:2]: Did you edit the YAML file?

File: vscode/syntaxes/toit.tmLanguage.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
   "$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
   "name": "Toit",
-  "firstLineMatch": "^#!/.*\\btoitc?\\b|(^#!/usr/bin/env toitc?)",
+  "firstLineMatch": "^#!/.*\\btoit([.]run)?$",
```
Comment (kasperl) [inline vscode/syntaxes/toit.tmLanguage.json:5]: I guess we do not care about `jag run -d host`?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: vscode/syntaxes/toit.tmLanguage.json (hunk 2 above)
Comment (floitsch) [reply vscode/syntaxes/toit.tmLanguage.json:5]: I don't think we will see that often in `#!` lines.
We can always add it later.

File: vscode/syntaxes/toit.tmLanguage.json (hunk 1 above)
Comment (floitsch) [reply vscode/syntaxes/toit.tmLanguage.json:2]: ...
thanks. Updated the yaml file as well.

**Pull request merged**